pkgs/tools/networking/opensnitch/daemon.nix

```diff
@@ -32,21 +33,25 @@ buildGoModule rec {
 
   modRoot = "daemon";
 
+  vendorSha256 = "sha256-LMwQBFkHg1sWIUITLOX2FZi5QUfOivvrkcl9ELO3Trk=";
+
+  nativeBuildInputs = [ pkg-config makeWrapper ];
+
+  buildInputs = [ libnetfilter_queue libnfnetlink ];
```

Comment:
And those libraries also work with our nftables-based firewall?

Reply:
Tested it in a VM with nixos-shell and on my system and applications are blocked until explicitly allowed